client/public/table.json

```diff
@@ -0,0 +1,8 @@
+{
+  "UUID": "0",
+  "timer": "00:00:00",
+  "players": [
+    { "name": "Player1", "UUID": "1"},
+    { "name": "Player2", "UUID": "2"}
+  ]
+}
```

Comment:
Please do a research about es-lint and add it to the project. Also add it to your IDE to automatically apply these lint rules when saving.